lib/adal.js

```diff
@@ -22,29 +22,23 @@ var AuthenticationContext = (function () {
     'use strict';
 
     /**
-     * Config information
-     * @public
-     * @class Config
-     * @property {tenant}          Your target tenant
-     * @property {clientId}        Identifier assigned to your app by Azure Active Directory
-     * @property {redirectUri}     Endpoint at which you expect to receive tokens
-     * @property {instance}        Azure Active Directory Instance(default:https://login.microsoftonline.com/)
-     * @property {endpoints}       Collection of {Endpoint-ResourceId} used for autmatically attaching tokens in webApi calls
-     */
-
-    /**
-     * User information from idtoken.
-     *  @class User
-     *  @property {string} userName - username assigned from upn or email.
-     *  @property {object} profile - properties parsed from idtoken.
+     * Configuration options for Authentication Context.
+     * @class config
+     *  @property {string} tenant - Your target tenant.
+     *  @property {string} clientID - Client ID assigned to your app by Azure Active Directory.
+     *  @property {string} redirectUri - Endpoint at which you expect to receive tokens. Set this value only if you want to redirect user to a page other than the login start page after successful authentication.
```

Comment:
The second half is not correct. Even if this is set, user will be redirected to login start page. Also, specify the default values wherever applicable. For example: instance, tenant, redirectUri, popUp.